### DISCUSSION TITLE
Page Structure Team is open to members

### DISCUSSION THREAD

**Discussion by JorgeRivera94** [uprm-inso4101-2023-2024-s2/semester-project-personalfinancemanager#121] [General]
Teams have been restructured for our second milestone. The Page Structure team will work on implementing a notification system and maintaining and bettering the app's UI. If you are interested in joining our team, please respond below.

**Comment by Goya01** [comment]
> Teams have been restructured for our second milestone. The Page Structure team will work on implementing a notification system and maintaining and bettering the app's UI. If you are interested in joining our team, please respond below.

I'd like to join this team!

**Reply by JorgeRivera94** [reply]
Hi! Ping me on discord (kfalling) to share contact info and availability for meetings.

**Comment by DavidCuevas3** [comment]
Im interested in joining the team

**Reply by JorgeRivera94** [reply]
Hi! Ping me on discord (kfalling) to share contact info and availability for meetings.

**Comment by Joaquin939** [comment]
I'd also like to join this team

**Comment by ianramospr** [comment]
> Teams have been restructured for our second milestone. The Page Structure team will work on implementing a notification system and maintaining and bettering the app's UI. If you are interested in joining our team, please respond below.

Im interested in joining the team
